src/map.cpp

```diff
@@ -1007,7 +1007,7 @@ void map::unboard_vehicle( const tripoint &p, bool dead_passenger )
     unboard_vehicle( *vp, passenger, dead_passenger );
 }
 
-vehicle *map::displace_vehicle( tripoint &p, const tripoint &dp )
+bool map::displace_vehicle( vehicle *veh, tripoint &p, const tripoint &dp )
```

Comment:
Would be better to accept a `vehicle` reference. Displacing a nullpointer does not make any sense and should not be allowed.

Reply:
Done, but I left other functions that uses vehicle pointer intact: like  `update_vehicle_cache` and `setremoteveh`. 
Changing them all could create a chain reaction. Should I do it anyway?